sys/src/9/amd64/sdiahci.c

```diff
@@ -1993,16 +1993,17 @@ didtype(Pcidev *p)
 		/*
 		 * 0x27c4 is the intel 82801 in compatibility (not sata) mode.
 		 */
-		if (p->did == 0x1e02 ||			/* c210 */
-		    p->did == 0x24d1 ||			/* 82801eb/er */
+		if (p->did == 0x1e02 ||				/* c210 */
```

Comment:
just wondering, what's with the whitespace change? it makes it harder to see what matters for i219 and what is just a whitespace change? 

generally, it is preferable to push 2 PRs for 2 different pieces of hardware. It makes it easier a few years down the line. I'm wondering if you would mind separating this into 2.